package.json

```diff
@@ -51,6 +51,6 @@
     "bignumber.js": "^6.0.0",
     "typed-promisify": "^0.4.0",
     "web3-provider-engine": "^13.6.5",
-    "wyvern-js": "git+https://github.com/ProjectOpenSea/wyvern-js.git#v3.3.2"
+    "wyvern-js": "git+https://github.com/ProjectOpenSea/wyvern-js.git#adb0b5ceb8616ce1414096a27c0a8a953120c0aa"
```

Comment:
can we use version number here, and also update this package to version 0.15?

Reply:
Will update to 0.15